docs/_includes/ww-admin-tasks-content.md

```diff
@@ -0,0 +1,250 @@
+as staff; optionally, one or more of ‘Create reports/updates on a user's behalf’, ‘Create reports/updates as anonymous user’ and ‘Create reports/updates as the council’ can be ticked.</span>
+
+Staff users are able to cancel subscriptions on behalf of residents in the same way the resident would do this - see The Resident’s experience. 
+
+**The process for setting up, renewing or cancelling subscriptions for users will differ depending on the payment provider you are using.** 
+
+</div>
+
+<div class="admin-task" markdown="1" id="creating-response-templates">
+
+### Creating and editing response templates
+
+<span class="admin-task__permissions">Permissions required: User must be marked as staff; “Add/edit response templates” must be ticked..</span>
+
+Staff users with the appropriate permissions may create and edit pre-written responses, so that they can be applied to reports quickly and easily.
+
+<img alt="Checking property collection details" src="/assets/img/ww-user-guide/WasteWorks-response-templates.png" />
+
+#### Creating a template
+
+Click on ‘Templates’ in the admin menu. You will see a table of any existing templates. Click on the button below, marked ‘New template’.
+
+Common template responses deal with statuses such as: Report received; Collection scheduled; Container ordered; Report under investigation.
+
+There is no limit to the number of template responses you can create. When creating templates, you should consider:
+
+What are the most common response types that you send back to report makers? It may be worth examining recent responses to find out.
+How should you word your response template text to make it as easy as possible for the report maker to understand the status of their issue?
+What title should you give the template so that it can be easily identified from a drop-down list when it is being applied by staff users?
+Input a title and some text for your template, then set which categories it should apply to. You may create different templates for each category, or use the same templates across multiple categories.
+
+You can also set a State so that the template response is automatically used when that state is applied to a report.
+
+For example, you can set it so that when a staff user changes a report’s state to ‘No further action’, the ‘Resolved’ status update text is automatically applied. While this functionality can be a time-saver, we advise using it with caution to ensure that the template text is applicable to every situation in which is will be automatically applied.
+
+If you have an Open311 connection, you can click ‘auto-response’ so that a template will be applied when the state is updated by the automated Open311 process. In this instance, if your Open311 server returns extra text as part of the update, you may put the placeholder {{description}} in the template here, and that placeholder will be replaced by the text from the Open311 server.
```

Comment:
Open311 not relevant to WW admin - would change to backend integration (and they will have to).